magento/pwa-studio
# [Doc] Move pagebuilder docs out of drafts

## Description

The PR moves the pagebuilder docs out of drafts and cherry picks changes from #2097 that were reverted in the release branch.

## Related Issue

Closes PWA-316

## Acceptance 

@bdenham

### Verification Stakeholders

@bdenham

### Specification



### Verification Steps

1. Build docs site and start preview server
2. Navigate to preview site
3. Verify new "Integrations" item in top nav that lists "Page Builder"
4. Navigate to Page Builder section
5. Verify existence of docs

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
